<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_info</class>
 <widget class="QDialog" name="register_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(238, 0, 0, 128), stop:1 rgba(238, 44, 44, 255));
}
QPushButton {
	border-radius:0px;
color: rgb(102, 160, 207);
	background-color:rgba(0,0,0,0);
	border-style:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #5CACEE;
	border-radius: 5px;
	padding: 2px;
	background: none;
	selection-background-color: #1B89CA;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #5CACEE;
	border-radius: 5px;
	padding: 2px;
	background: none;
	selection-background-color: #1B89CA;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_id_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #5CACEE;
	border-radius: 5px;
	padding: 2px;
	background: none;
	selection-background-color: #1B89CA;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_id_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:10px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
